src/alto-ui/helpers/number.js

```diff
@@ -18,6 +18,7 @@ const decimalSeparatorByLocale = {
 const DEFAULT_DECIMAL_SEPARATOR = '.';
 
 export const round = precision => n => Math.round(n * 10 ** precision) / 10 ** precision;
+export const percentRound = precision => n => parseFloat(n.toFixed(precision));
```

Comment:
```js
export const round = precision => n => parseFloat((Math.round(n * 10 ** precision) / 10 ** precision).toFixed(precision));
```

Reply:
Updated
used in a slightly changed form, to dont confuse other devs who jump into this,